README.md

```diff
@@ -19,53 +19,105 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
 
 Thimble requires a modern web browser, and we recommend using Mozilla Firefox or Google Chrome.
 
-Setup
------
+Thimble interacts with the Publish API (source managed in [publish.webmaker.org](https://github.com/mozilla/publish.webmaker.org)) to store users, projects, files and other content as well as publish user projects.
+
+For authentication and user management, Thimble uses Webmaker OAuth which consists of the Webmaker ID System (source managed in [id.webmaker.org](htps://github.com/mozilla/id.webmaker.org)) and the Webmaker Login API (source managed in [login.webmaker.org](https://github.com/mozilla/login.webmaker.org)).
+
+All three services along with Thimble are bundled together using Git subtrees and run together using Vagrant.
+
+Setup/Installation
+--------
 
-### Requirements
+## 1) Pre-Requisites
+In order for Thimble to be installed correctly, the following dependencies needs to be installed:
 
+- Brackets (Bramble) [[download](https://github.com/mozilla/brackets)]
 - Node.js (version 4.6 or later) [[download](https://nodejs.org/en/download/)]
 - Virtualbox (version 5.1 or later) [[download](https://www.virtualbox.org/wiki/Downloads)]
 - Vagrant (version 1.9 or later) [[download](https://www.vagrantup.com/downloads.html)]
-  - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable
+  - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable.
 
-The setup of Thimble can be divided into two distinct sections:
+### Installing Brackets (Bramble)
+* Fork and clone https://github.com/mozilla/brackets
```

Comment:
I honestly think that the instructions in https://github.com/mozilla/thimble.mozilla.org/pull/1683/files#diff-04c6e90faac2675aa89e2176d2eec7d8L38 (and below) to install Brackets are a bit clearer so I would undo this change. (I still need to update the Wiki with the correct instructions for manual install)